core-aam/aria-valuenow-spinbutton-default-manual.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-valuenow</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "result",
+                  "atk_value_get_current_value()",
+                  "is",
+                  "0"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "AXValue",
+                  "is",
+                  ""
+               ]
+            ],
+            "IAccessible2" : [
+               [
+                  "property",
+                  "currentValue",
+                  "is",
+                  "VT_EMPTY"
```

Comment:
I'm not sure this is a valid test, but it looks like there is support for a value to have "no value" already in Chrome for this API: https://source.chromium.org/chromium/chromium/src/+/main:ui/accessibility/platform/ax_platform_node_win.cc;l=4580?q=VT_EMPTY%20f:_win.cc